src/web-stack/apps/positive-affirmations/src/app/users/entities/affirmation.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { UserEntity } from './user.entity'
+
+@Entity({ name: 'user' })
```

Comment:
This entity should be named 'affirmation'

Reply:
Alright that makes sense. This is really cool, btw!! Love it